lib/job_processor.js

```diff
@@ -108,6 +108,7 @@ module.exports = function (log) {
       log.info('Compressing project:', project.name);
       var start = Date.now();
 
+      var progressFilepath = path.replace('.tar.gz', '_inprogress.tar.gz');
       var output = fs.createWriteStream(path);
```

Comment:
you write to `path`, so when you try to rename `progressFilepath` to `path` (L 121), file doesn't exist. It should be `fs.createWriteStream(progressFilepath)`